test/support/builders/atom_watcher_event.js

```diff
@@ -91,6 +91,11 @@ module.exports = class AtomWatcherEventBuilder {
     return this
   }
 
+  updatedAt (mtime /*: Date */) /*: this */ {
```

Comment:
Should we rename this method to `mtime`?

`AtomWatcherEvent` type [doesn't have an `updatedAt` property](https://github.com/cozy-labs/cozy-desktop/blob/v3.13.0-alpha.1/core/local/steps/event.js#L19-L29).
And `Metadata#updatedAt` can be set to [either mtime or ctime](https://github.com/cozy-labs/cozy-desktop/blob/v3.13.0-alpha.1/core/metadata.js#L473) (depending on whether the file metadata or content were modified).

So I believe `#updatedAt` may be confusing here.